doc/source/release-notes/Percona-Server-5.5.52-38.3.rst

```diff
@@ -0,0 +1,53 @@
+.. rn:: 5.5.52-38.3
+
+============================
+|Percona Server| 5.5.52-38.3
+============================
+
+Percona is glad to announce the release of |Percona Server| 5.5.52-38.3 on
+October 4th, 2016. Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.5/Percona-Server-5.5.52-38.3/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+
+Based on `MySQL 5.5.52
+<http://dev.mysql.com/doc/relnotes/mysql/5.5/en/news-5-5-52.html>`_, including
+all the bug fixes in it, |Percona Server| 5.5.52-38.3 is now the current stable
+release in the 5.5 series. All of |Percona|'s software is open-source and free,
+all the details of the release can be found in the `5.5.52-38.3 milestone at
+Launchpad <https://launchpad.net/percona-server/+milestone/5.5.52-38.3>`_.
+
+Bugs Fixed
+==========
+
+ ``mysql_upgrade`` would fail when server was running with
```

Comment:
mysql_upgrade now does not binlog its actions by default. To restore the previous behavior, use --write-binlog option. Bug fixed ... s/14221043/1065841